src/main/kotlin/net/atos/zac/mail/MailService.kt

```diff
@@ -98,6 +128,7 @@ class MailService @Inject constructor(
         val replyToAddress = mailGegevens.replyTo?.toAddress().let {
             if (fromAddress == it) null else it
         }
+        setupPasswordAuthentication()
```

Comment:
Like @bas-info-nl also mentioned it seems a bit weird to explicitly call such a setup method indeed. There must be a nicer way to do this I would think?